src/Opaleye/SqlTypes.hs

```diff
@@ -170,6 +173,8 @@ sqlTimeOfDay = P.pgTimeOfDay
 -- "We recommend not using the type time with time zone"
 -- http://www.postgresql.org/docs/8.3/static/datatype-datetime.html
 
+sqlInterval :: Time.CalendarDiffTime -> F.Field SqlInterval
+sqlInterval = P.pgInterval
```

Comment:
Then we can just re-export `sqlInterval` rather than making up a new name for it.